### PR TITLE
Prepare 2022.09

### PR DESCRIPTION
**Pull request by juliogonzalez**
## What does this PR change?

Prepare 2022.09:
- Missing changelogs
- Add a missing migration path for the reportdb

## GUI diff

No difference.

Before:

After:

- [x] **DONE**

## Documentation
- No documentation needed: Changelogs/db migration path

- [x] **DONE**

## Test coverage
- No tests: Tested by OBS/testsuite

- [x] **DONE**

## Links

Related: https://github.com/SUSE/spacewalk/issues/18491

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: containers/proxy-helm/proxy-helm.changes [hunk 1]
```diff
@@ -1,5 +1,9 @@
-- Remove the namespace configuration value to honor helm's -n
-- Add the log level value to the config.yaml
+-------------------------------------------------------------------
+Wed Sep 28 07:43:59 UTC 2022 - Julio González Gil <jgonzalez@suse.com>
+
+- Update the image version to 2022.09:
```
[the remark this answers is not in the document]
Comment (deneb-alpha) [reply containers/proxy-helm/proxy-helm.changes:4]:
```suggestion
- Update the chart version to 2022.09:
```

**Review by deneb-alpha — COMMENTED** [review]
on the proxy-helm chart I think it should be better to say that the update is for the chart, not for an image.

File: containers/proxy-helm/proxy-helm.changes (hunk 1 above)
[the remark this answers is not in the document]
Comment (juliogonzalez) [reply containers/proxy-helm/proxy-helm.changes]: Of course, nice catch :-D

**Review by deneb-alpha — APPROVED** [review]
LGTM :)

**Pull request merged**